spec/spec.js

```diff
@@ -0,0 +1,96 @@
+describe('type-of-is', function(){
```

Comment:
Yes,  with console.log you can get the result of every statements, but you don't know whether the result is what you want. 

And with jasmine, every result will be compare with the result you want automatically.
![image](https://cloud.githubusercontent.com/assets/2059860/15035410/321f8bf6-12b4-11e6-8379-1c1c484c8448.png)

I think this is really important for the stable of this project.